keyboards/akb/raine/config.h

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2019 Elliot Powell
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
```

Comment:
```suggestion
#pragma once

```

Reply:
Unless there is a specific reason against it, we want to have `#pragma once` here.  There are some edge cases where not having it can cause issues.